Makefile: complement CFLAGS provided by the user

In some circumstances, the user may want to pass its own CFLAGS,
like for when the zlib headers are not located in the standard gcc
search paths.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

diff -durN a/mkfs.axfs-legacy/Makefile b/mkfs.axfs-legacy/Makefile
--- a/mkfs.axfs-legacy/Makefile
+++ b/mkfs.axfs-legacy/Makefile
@@ -1,5 +1,5 @@
-INC = -I./
-CFLAGS = -g $(INC) -O0
+INC += -I./
+CFLAGS += -g $(INC) -O0
 
 MKFSOBJS = mkfs.axfs.o
 
